WordPress/src/main/java/org/wordpress/android/ui/mediapicker/MediaPickerFragment.kt

```diff
@@ -499,7 +499,8 @@ class MediaPickerFragment : Fragment() {
     private fun setupAdapter(items: List<MediaPickerUiItem>) {
         if (recycler.adapter == null) {
             recycler.adapter = MediaPickerAdapter(
-                    imageManager
+                    imageManager,
+                    viewModel
```

Comment:
🔍 Nitpick: We might want to consider casting the VM so the code is more readable ` viewModel as CoroutineScope`. It scared me at first that we are passing the VM into the adapter, before I realized we are passing just the scope.

Reply:
Good idea 👍 , casted in e30225ad41